src/quo2/components/wallet/network_amount/view.cljs

```diff
@@ -0,0 +1,22 @@
+(ns quo2.components.wallet.network-amount.view
+  (:require
+    [clojure.string :as str]
+    [quo2.components.markdown.text :as text]
+    [react-native.core :as rn]
+    [status-im2.common.resources :as resources]
+    [quo2.components.wallet.network-amount.style :as style]))
+
+(defn network-amount
+  [{:keys [amount token]}]
+  [rn/view {:style style/container}
+   [rn/image
+    {:source (resources/tokens token)
+     :style  {:width 12 :height 12}}]
+   [text/text
+    {:width :medium
+     :size  :paragraph-2
+     :style style/text}
+    (str amount " " (str/upper-case (clj->js token)))]
```

Comment:
do we know tokens are always upper-case? also I don't think we should require our tags resource library names to match the tokens name.
maybe it's better we just add another prop and add value ourselves?

```
(defn network-amount
  [{:keys [amount token-source]} value]
  [rn/view {:style style/container}
   [rn/image
    {:source (resources/tokens token-source)
     :style  {:width 12 :height 12}}]
   [text/text
    {:width :medium
     :size  :paragraph-2
     :style style/text}
    value)]
    ```
    and nit: perhaps `token-source` is more accurate?

Reply:
@J-Son89 I think here we need to write the 3 letter code of a token in upper case as shown in the design https://www.figma.com/file/WQZcp6S0EnzxdTL4taoKDv/Design-System-for-Mobile?type=design&node-id=919-27533&mode=design&t=SUJdrjvvqmlVNnrC-0

Reply:
sure, have you confirmed this with design? of course if it is always upper case then we should leave as is 👍

Reply:
@J-Son89 Yes, confirmed, always captial.

Reply:
Great stuff @OmarBasem. Nice one for checking!